std/protocols/lookup.asm

```diff
@@ -75,6 +74,7 @@ let lookup: Constr, expr -> () = constr |lookup_constraint, multiplicities| {
 
     let lhs_denom = sub_ext(beta, fingerprint(lhs, alpha));
     let rhs_denom = sub_ext(beta, fingerprint(rhs, alpha));
+    let multiplicities = std::prover::new_witness_col_at_stage("multiplicities", 0);
```

Comment:
Why specify the stage explicitly?

Reply:
If you use `let multiplicities`, it will be create at the "current stage".

Reply:
Right, if we use `capture_stage`, then we might already be at a higher stage, but I think there should still be a better way around that than just setting the stage to zero explicitly. I think I would just use `let multiplicites` now until we reach that problem.

Reply:
done